transports/dns/src/lib.rs

```diff
@@ -160,13 +160,13 @@ use async_trait::async_trait;
 use futures::{future::BoxFuture, prelude::*};
 pub use hickory_resolver::{
     config::{ResolverConfig, ResolverOpts},
-    error::{ResolveError, ResolveErrorKind},
+    {ResolveError, ResolveErrorKind},
 };
 use hickory_resolver::{
     lookup::{Ipv4Lookup, Ipv6Lookup, TxtLookup},
     lookup_ip::LookupIp,
     name_server::ConnectionProvider,
-    AsyncResolver,
+    Resolver as AsyncResolver,
```

Comment:
Nit: Slight preference to call it `HickoryResolver` instead, to avoid confusion with the actual, now deprecated `hickory_resolver::AsyncResolver`.

Alternatively, since it only has one occurrence we could also just use `hickory_resolver::Resolver` there directly.

Reply:
Fixed.